src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -53,12 +70,6 @@ def pivot_categorical(
 ##########################
 
 
-def get_distribution_data(builder, risk: "Risk") -> Dict[str, Any]:
-    validate_distribution_data_source(builder, risk.risk)
```

Comment:
Do you still need `validate_distribution_data_source`? This is the only place I can see where it was used.